integration-tests/hibernate-orm-tenancy/discriminator/src/main/resources/db/migration/V1.0.0__init.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE known_fruits
```

Comment:
I would remove this file as it's not strictly necessary in this case.

Reply:
ok i will just use simple import.sql